microprofile/opentracing-extension/src/main/java/org/wildfly/extension/microprofile/opentracing/TracingExtensionLogger.java

```diff
@@ -43,18 +43,10 @@
     @Message(id = 2, value = "MicroProfile OpenTracing Subsystem is processing deployment")
     void processingDeployment();
 
-    @LogMessage(level = DEBUG)
-    @Message(id = 3, value = "Registering SmallRye CDI Extension")
-    void registeringCDIExtension();
-
     @LogMessage(level = DEBUG)
     @Message(id = 4, value = "The deployment does not have CDI enabled. Skipping MicroProfile OpenTracing integration.")
```

Comment:
@jpkrohling As this has not yet been part of a proper release, it is better to just 'compress' the log ids. i.e. make this one 3 and so on. (If it had been part of a release, then the recommended way to remove unused log messages is to comment them out rather than remove them)

Reply:
What's considered a proper release? I think the first release this has been a part of is 14.0.0.Beta2.

Reply:
@jpkrohling I don't think we need to care about the Beta. The important is the Final which ends up in a localized product. But if you want to be extra safe, you can go with the comment out option

Reply:
Message IDs updated!